lib.php

```diff
@@ -33,8 +33,9 @@
  * @since  Moodle 3.3
  */
 function tool_seo_before_standard_html_head() {
+    global $PAGE;
 
-    if (tool_seo_is_current_url_excluded()) {
+    if (tool_seo_is_url_excluded($PAGE->url->get_path())) {
```

Comment:
I think this if is the wrong way around?

Reply:
I think the 'noindex' and 'excluded' negatives get confusing. I'll look into rewording the functions. Basically, this condition checks if the URL is in the list of excluded URLs and if true, does not add the noindex meta tag.

Reply:
The default is that the meta tag is included.

Reply:
The logic is still the wrong way around. By default we should not add the meta tag and only add the meta which excludes the page from index, when it matches the exclusion list.

I think there is actually a moodle rule of thumb that variables and methods should generally be positive so to clarify that maybe rename it to 'tool_seo_is_indexable'. I don't like '_is_indexed' because that it implies it is actually indexed which isn't the case.